.markdownlint.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can it check columns wide?

Reply:
Yes! Its default lint rule is that max line width = 80.